src/mrgw.sh

```diff
@@ -11,7 +11,7 @@ port=$2
 
 shift 2
 
-run_root=$script_root/run/$name
+run_root=run/$name
```

Comment:
@cbodley does it mean that running mrgw.sh from different locations will end up with different instances?